src/cache/age.rs

```diff
@@ -62,7 +62,7 @@ impl Age {
         // entry. We want the last entry.
         let header = headers.iter().last().unwrap();
 
-        let num: u64 = header.as_str().parse().status(400)?;
+        let num: u64 = header.as_str().parse::<u64>().status(400)?;
```

Comment:
@pbzweihander Did these suddenly become required?

Reply:
Yes. But I don't know exactly why...

Reply:
I think it's because `.status()` uses `TryInto` inference but `?` also uses similar inference.

I still think this is probably a good idea (at least until `http-types::Error` can impl `StdError`), but it means this would have to happen in 3.0.